src/cli.ts

```diff
@@ -41,9 +41,12 @@ import { run } from './';
 const program = parseArgs();
 const resolvedCwd = cwd();
 /**
- * Set debug based on flag
+ * Set debug based on DEBUG ENV and -d flags
+ * namespsace - es - elastic synthetics
```

Comment:
`s/namespsace/namespace/`

Maybe we should make the namespace `synthetics`? `es` sounds like Elasticsearch for me, and is more common within our org. Maybe `synthetics` is more appropriate? Longer, I know.

Also, can we document this somewhere? This could be a separate PR as well.

Reply:
Ahh, i was going for `Elastic synthetics` as `es` and didnt even think of Elasticsearch :D 

Yeah `synthetics` or `syn` for short works as well?